browser/brave_rewards/extension_rewards_service_observer.cc

```diff
@@ -34,13 +34,38 @@ void ExtensionRewardsServiceObserver::OnWalletInitialized(
     return;
   }
 
+  // ledger::Result::WALLET_CREATED
+  if (result == 12) {
```

Comment:
Even if we have to mirror this ledger enum on the rewards side (with a comment at the top mentioning that they have to be kept in sync, etc.), I think it'd be preferable to this. 

If we wanted to get really fancy, we could probably have a build step that parses and generates the enum on the rewards side.

Reply:
@emerick can you please log an issue for that as this is how all enums are handled in observers right now. There is a plan to allow ledger into observers as well so that we can use ledger directly

Reply:
Will do.